clouddriver-ecs/src/integration/README.md

```diff
@@ -8,3 +8,22 @@ To manually run the gradle task for these tests:
 ```bash
 $> ./gradlew :clouddriver-ecs:integrationTest
 ```
+
+## Guidance for modifying this package
+
+### Adding new tests
+
+New Amazon ECS provider features of significant scope should include an integration test which exercises new functionality.
+Examples of qualifying changes include (but are not limited to):
+
+* Implementing a new atomic operation
+* New forking logic in how `CreateServerGroup` functions or is validated (especially re: broadly impactful settings like `networkMode`, launch type, load balancing, or application autoscaling)
+* Adding a new controller for a new type of resource
+
```

Comment:
Can we mention about mocking of some of the main clients like AmazonClientProvider and AmazonElasticLoadBalancing.  Also, preparing the JSON file for inputs and artifacts with significant values.

Reply:
So this section is supposed to answer the question: "what kinds of new features require an integration test?" - so how to mock required clients and request payloads/responses isn't quite relevant here, but if you think we could use a section in this doc about how to author a test case, that would be a good addition to make in a future PR

Reply:
actually i can see that the section title isn't super clear re: this intention so i will update